src/cljs/main/broadfcui/page/method_repo/method_config_importer.cljs

```diff
@@ -47,7 +47,10 @@
                        (swap! state assoc :error (get-parsed-response false))))})))})
 
 (defn- create-import-form [state props this locals entity config? fields]
-  (let [{:keys [workspace-id]} props]
+  (let [{:keys [workspace-id]} props
+         currentEmail (-> @utils/auth2-atom (.-currentUser) (.get) (.getBasicProfile) (.getEmail))
+         owners (set (:managers entity))
+         owner? (contains? owners currentEmail)]
```

Comment:
In my testing, it seems that replacing lines 52 and 53 with `owner? (contains? (:managers entity) currentEmail)]` is functionally equivalent (skip creating `owners` variable, skip `set` because `contains?` also works on maps).